src/libraries/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.WriteValues.Raw.cs

```diff
@@ -0,0 +1,179 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+
+namespace System.Text.Json
+{
+    public sealed partial class Utf8JsonWriter
+    {
+        /// <summary>
+        /// Writes the input as JSON content. It is expected that the input content is a single complete JSON value.
+        /// </summary>
+        /// <param name="json">The raw JSON content to write.</param>
+        /// <param name="skipInputValidation">Whether to validate if the input is an RFC 8259-compliant JSON payload.</param>
+        /// <exception cref="ArgumentNullException">Thrown if <paramref name="json"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException">Thrown if the length of the input is zero or greater than 715,827,882.</exception>
+        /// <exception cref="JsonException">Thrown if <paramref name="skipInputValidation"/> is <see langword="false"/>, and the input is not  RFC 8259-compliant.</exception>
```

Comment:
For the uninitiated, it'd be best to also state what that implies.

And I wonder if stating the specific RFC number is needed (a new version could come out and we'd have to update the docs). We could follow the Utf8JsonReader docs and just provide a link:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader?view=net-5.0#remarks

The `Read()` method docs just say JSON RFC:
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.utf8jsonreader.read?view=net-5.0#exceptions

Another case where it could throw (add a test), is JSON containing too deep nesting.

```suggestion
        /// <exception cref="JsonException">Thrown if <paramref name="skipInputValidation"/> is <see langword="false"/>, and the input is not a valid, complete, single JSON value according to the JSON RFC (https://tools.ietf.org/html/rfc8259) or the input JSON exceeds the default recursive depth of 64.</exception>
```

Reply:
Makes sense. Added a test for too deep nesting 👍